sklearn/tree/_utils.pyx

```diff
@@ -611,9 +597,9 @@ cdef class WeightedMedianCalculator:
                                                   original_median)
         return return_value
 
-    cdef int update_median_parameters_post_remove(self, DOUBLE_t data,
-                                                  DOUBLE_t weight,
-                                                  double original_median) nogil:
```

Comment:
No need to change your PR, but please try to avoid introducing formatting changes on lines that are not related to the PR itself in the future. In this case I think that both indentation styles are valid so I don't really see the point.